src/tabs.js

```diff
@@ -196,7 +196,7 @@ function tabsFactory(container, config) {
         }
 
         if (component.getConfig().hideLoneTab && tabs.length === 1) {
-            hider.hide(component.getElement().find(tabSelector));
+            hider.hide(component.getElement().parent());
```

Comment:
**issue:** This is harmful to the container. Except for very particular cases, a component must not try to modify its container. In this particular case, what will happen if the tabs are living aside from other components? All will be hidden for no reason.

**suggestion:** Please consider hiding the component itself. Do not forget to remove the unused resource import for `'ui/hider'` too.
```suggestion
            component.hide();
```